docs/reference/projectyml.rst

```diff
@@ -54,6 +54,67 @@ The code snippet below provides an example of a complete ``project.yml`` file:
       index_urls:
         - https://pypi.org/simple/
 
+
+.. _specify_location_pip:
+
+
+Specify locations from where V2 modules will be installed
+---------------------------------------------------------
+This section explains how to configure your project in order to download v2 modules from any python package repository.
+By default, a project created using the :ref:`project-creation-guide` is configured to install packages from ``https://pypi.org/simple/``. There are multiple ways
+to change this behaviour.
+
+Using pip config file at a standard location
+""""""""""""""""""""""""""""""""
+
+By setting the ``use_config_file`` option of the pip section to ``True``, the project will use the pip config files.
+
+.. code-block:: yaml
+
+    pip:
+      use_config_file: True
+
+To specify the url of a pip repository, add the following to the pip config file:
+
+.. code-block:: text
+
+  [global]
+  timeout = 60
+  index-url = <url of a python package repository>
+  extra-index-url = <url of a second python package repository>
+                    <url of a third python package repository>
+
+Creating a pip config file at a custom location
+""""""""""""""""""""""""""""""
+
+Alternatively, a pip config file can be used at a custom location.
+The ``index-url`` can be specified in this file as explained in the previous section.
+To make this work, the ``PIP_CONFIG_FILE`` environment variable needs to be set to the path of the newly created file (See: :ref:`env_vars`).
```

Comment:
Wouldn't it be easier to just refer to the appropriate page on the pip documentation?

Reply:
I added the link to the docs but I didn't remove anything. I find it relevant to refer to our env_vars doc as I didn't now how to set them when I did the testing.